### PR TITLE
[cmd:load] Add option to clear ids relationships and enrollments

### LINKED ISSUE #94: Reset unique identities before loading
The command `load`appends the data from the given file to the data stored in the database. Sometimes, we need to have exactly the same relationships between identities that we have on the input file.

The idea is to have a flag named `--reset` that will remove all the identities assigned to the unique identities. Identities will be moved to a single unique identity with the same id. In other words, 1:N relationships will be removed ending up with 1:1.

Information about enrollments must be removed too.

### PR DESCRIPTION
**Pull request by sduenas**
Before loading any data, if 'reset' option is set, all the
relationships between identities and their enrollments will
be removed from the database.

Thus, both input and database will have the same status for
the loaded unique identities.

This commit fixes #94

### CODE REVIEW

File: sortinghat/cmd/load.py [hunk 1]
```diff
@@ -49,6 +49,9 @@ class Load(Command):
     engine is selected. The parameter '--match-new' can also be used to match
     only those identities that are new on the registry.
 
+    Previous relationships between identities and their enrollments will be
```
Comment (acs) [inline sortinghat/cmd/load.py]: «Previous relationships between identities and their enrollments will be removed when the option '--reset' is set.»

I would reword it to:

«If the the option '--reset' is set, previous relationships between identities and their enrollments will be removed»

**Comment by sduenas** [comment]
For big databases it will last a little longer but I hope it won't be a problem.
